src/tasks/noncombat.ts

```diff
@@ -94,6 +94,9 @@ export const NoncombatQuest: Quest = {
           $effect`Throwing Some Shade`,
           $effect`Ultra-Soft Steps`,
 
+          // eslint-disable-next-line libram/verify-constants
+          $effect`Hiding from Seekers`,
```

Comment:
We should also update `computeCombatFrequency()` once we know the exact -combat% value this provides